docs/development/schema.md

```diff
@@ -1,75 +1,136 @@
-# Schema development
+# Data modelling and schema development
 
-This section outlines a number of the pattens we commonly use to develop the schema for an open data standard.
+This page provides an overview of the steps involved in data modelling and schema development.
 
-```{admonition} Learning / reflection
----
-class: note
----
-We currently jump straight from the conceptual framework document, to working up the data model and schema for a standard in a schema language. 
+## Document a data model
 
-This differs [from the approach proposed here](https://github.com/open-contracting-archive/technical-approach#data-model) of maintaining the **data model** as a narrative document, only then given form by a schema as a reference implementation. 
+A data model is an abstract model that organizes elements of data and standardises how they relate to one another and to the properties of real-world entities. A data model focuses on what data represents rather than how it is stored or exchanged.
+
+Before authoring the schema for a standard and committing to specific implementation details, it is recommended to document a data mode to help stakeholders align on definitions and relationships.
+
+The data model for a standard should be based on its [research](research.md) and conceptual framework. Documenting the data model for a standard involves identifying and defining the entities (classes), attributes (properties), relationships and permissable values (codelists) needed to satisfy the requirements, user stories and use cases for the standard.
```

Comment:
```suggestion
The data model for a standard should be based on [research](research.md) into the related policy-area and a thorough understanding of the concepts which underpin it (a conceptual framework). Documenting the data model for a standard involves identifying and defining the entities (classes), attributes (properties), relationships and permissable values (codelists) needed to satisfy the requirements, user stories and use cases for the standard.
```
I think 'policy area' fits better with the text elsewhere. Otherwise, 'domain'.